__init__.py

```diff
@@ -191,7 +192,10 @@ def _start_new_timer(self, message):
         else:
             timer = self._build_timer(duration, name)
             self.active_timers.append(timer)
-            self.active_timers.sort(key=lambda x: x.expiration)
+            self.active_timers.sort(key=lambda tmr: tmr.expiration)
+            if len(self.active_timers) == 1:
+                self._show_gui()
+                self._start_display_update()
             self._speak_new_timer(timer)
             self._save_timers()
 
```

Comment:
In assign_timer_name() why don't you just always name the first timer "Timer 1"? Why do you want the exception case?

Reply:
This was a direction from Derick.  He didn't want the timers numbered unless there was more than one.